src/TuDa.CIMS.Api/ConfigureServices.cs

```diff
@@ -11,6 +11,7 @@ public static WebApplication SetupScalar(this WebApplication app)
     {
         app.MapScalarApiReference(options =>
         {
+            options.Servers = [];
```

Comment:
🌱 Maybe add a comment, that this is needed to clear the cashed IPs to work with Aspire